### PR TITLE
Move non verified alert outside of user profile header

### PR DESCRIPTION
**Pull request by daksas303**
- Moved alert to be outside of profile header and appear according to Figma. 
- Changed padding of alert text on mobile
- https://github.com/bratislava/konto.bratislava.sk/issues/681

### CODE REVIEW

File: next/components/forms/segments/UserProfile/UserProfileDetail.tsx [hunk 1]
```diff
@@ -45,10 +47,28 @@ const UserProfileDetail = (props: UserProfileDetailProps) => {
 
   return (
     <div
-      className={cx(' flex flex-col', 'md:static md:z-0', {
+      className={cx('mt-3 flex flex-col bg-white', 'md:static md:z-0', {
```
Comment (mpinter) [inline next/components/forms/segments/UserProfile/UserProfileDetail.tsx:50]: the mt-3 is creating this gray-ish block on top (see screenshot) - macOs chrome

Screenshot 2024-01-04 at 10 34 38

File: next/components/forms/info-components/Alert.tsx [hunk 2]
```diff
@@ -111,15 +111,15 @@ const Alert = ({
...
             'text-gray-0': solid,
             'text-gray-700': !solid,
           })}
         >
           {title && message}
         </div>
       )}
-      <AlertButtons className="pl-9" buttons={buttons} />
+      <AlertButtons className="md:pl-9" buttons={buttons} />
     </div>
```
Comment (mpinter) [inline next/components/forms/info-components/Alert.tsx:123]: I see where this is coming from but that's actually an issue with out figma, the component has it correclty, it's wrong in the designs. 

I've contanted relevant people to fix the figma side. Please revert the changes here.

**Review by mpinter — CHANGES_REQUESTED** [review]

File: next/components/forms/info-components/Alert.tsx (hunk 2 above)
Comment (daksas303) [reply next/components/forms/info-components/Alert.tsx]: Reverted

File: next/components/forms/segments/UserProfile/UserProfileDetail.tsx (hunk 1 above)
Comment (daksas303) [reply next/components/forms/segments/UserProfile/UserProfileDetail.tsx]: Changed margin to padding

**Review by mpinter — APPROVED** [review]
